cli/chaosli/README.md

```diff
@@ -13,6 +13,9 @@ The Chaos Controller CLI, `chaosli`, is meant to help create a more user friendl
 
 Download the `chaosli` binary for your operating system and architecture [from the latest release](https://github.com/DataDog/chaos-controller/releases/latest).
 
+##### Requirements (important):
```

Comment:
nit: I think `(important)` is redundant after "Requirements"

Reply:
Fair enough haha